google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -75,6 +79,8 @@
   private final Map<String, XBreakpoint> myIdeBreakpoints = new ConcurrentHashMap<String, XBreakpoint>();
   private final CloudDebugProcess myProcess;
   private PsiManager myPsiManager;
+  private VirtualFileSystem fileSystem =
```

Comment:
wouldn't you prefer to initialize this in the constructor and use constructor injection for tests?  Not sure what's involved in the VFM initialization and what the deps are.